app/controllers/application_controller.rb

```diff
@@ -2,11 +2,18 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
   rescue_from ActiveRecord::RecordNotFound, with: :not_found
+  rescue_from ActionController::UnknownFormat, with: :unknown_format
+
   def not_found
     flash[:notice] = I18n.t('flash.record_not_found')
     render(file: 'application/_missing',
            status: :not_found,
            formats: [:html],
-           layout:  true)
+           layout: true)
+  end
+
+  def unknown_format(error)
```

Comment:
Unused method argument - `error`. If it's necessary, use `_` or `_error` as an argument name to indicate that it won't be used. You can also write as `unknown_format(*)` if you want the method to accept any arguments but don't care about them.